src/main/java/org/opensearch/security/dlic/rest/api/InternalUsersApiAction.java

```diff
@@ -194,7 +194,7 @@ protected AbstractConfigurationValidator postProcessApplyPatchResult(RestChannel
                 builder.endObject();
                 retVal = getValidator(request, BytesReference.bytes(builder), resourceName);
             } catch (IOException e) {
-                log.error(e);
+                log.error(e.toString());
```

Comment:
I find the best way to handle logging exceptions to be:

```
log.error("relevant message", ex);
```

This approach will write out the stack trace.